README.md

```diff
@@ -41,7 +41,7 @@ After you export your ensemble, you reference the directory to run an instance o
 tritonserver --model-repository=/export_path/
 ```
 
-Refer to the [Merlin Systems Example Notebooks](http://github.io/NVIDIA-Merlin/systems/main/examples/README.html) for a notebook that serves a ranking models ensemble.
+Refer to the [Merlin Systems Example Notebooks](https://github.com/NVIDIA-Merlin/systems/tree/main/examples) for a notebook that serves a ranking models ensemble.
```

Comment:
We could simplify this link further to `./examples/` if we want the link to be consistent with the commit being viewed.